.cirrus.yml

```diff
@@ -1013,6 +1045,7 @@ success_task:
         # Label triggered task. If made automatic, remove line below
         # AND bypass in contrib/cirrus/cirrus_yaml_test.py for this name.
         # - podman_machine
+        # - podman_machine_aarch64
```

Comment:
Assuming "just run podman-machine always" works, please remember to uncomment these lines and also update `contrib/cirrus/cirrus_yaml_test.py` (remove the tasks from the exclude list).

Reply:
@cevich do we really want to spin up those huge-@$$ bare-metal instances for every PR?

Reply:
I don't see a way around that now, and maybe it's for the better (almost any PR could break podman-machine).  It could be limited to only running in PRs with an `only_if` condition: `$CIRRUS_PR != ''"`.  That's probably a good idea for the aarch64 podman-machine as well.

Reply:
@cevich updated. PTAL